tests/pkg_tomsfastmath/main.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2016 Mathias Tausig, FH Campus Wien
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <inttypes.h>
+#include <thread.h>
+
+#include "tfm.h"
+
+int small_tests(void)
```

Comment:
If you'd like to convey the information from the test's README, you could e.g. call this test `basic_arithmetic_test` instead? Just an idea though :)